Makefile

```diff
@@ -113,3 +107,71 @@ generate-pip-lockfiles: auth/pinned-requirements.txt
 generate-pip-lockfiles: batch/pinned-requirements.txt
 generate-pip-lockfiles: ci/pinned-requirements.txt
 generate-pip-lockfiles: memory/pinned-requirements.txt
+
+$(HAILTOP_VERSION):
+	$(MAKE) -C hail python/hailtop/hail_version
+
+hail-ubuntu-image: docker/hail-ubuntu
```

Comment:
Make doesn't do what you'd want when you specify a directory as a dependency. You'll need to use something like `$(shell git ls-files docker/hail-ubuntu)`.

Reply:
Ah ya I missed this one